css/src/components/form/help.scss

```diff
@@ -0,0 +1,23 @@
+$help-color: $text !default;
+$help-error-color: $danger-dark !default;
+$help-warning-color: $warning-dark !default;
```

Comment:
Should the warning color be a shade of yellow? On light theme, the warning color looks like a shade of gray to me 
![image](https://user-images.githubusercontent.com/94572161/153277437-19ba3049-3243-4e48-90b9-7ac643b6ff98.png)

Reply:
I see your point, however, we need to be able to pass the contrast accessibility tests, and the regular `$warning`, which is a brighter yellow color, doesn't meet the requirements.

Reply:
Also, shouldn't error be a shade of red? Same for input

Reply:
I see. I tried playing with the colors a bit. It looks to me that `$danger` should work with medium background in all themes except for `dark`. Please double check, but I think it would work if we update the dark `$danger-base` to be one shade lighter (`$palette-red-20`).

I'm just thinking if `$danger-base` or any state colors provide insufficient contrast, then perhaps we should update them so that we can actually use those variables. Otherwise, it means `$danger-base` can't realistically be used in most (if not all) cases and making it available to use seems misleading.

![image](https://user-images.githubusercontent.com/94572161/153288200-76a42396-1a21-41f2-9cb2-ce312a70e8dc.png)

Reply:
I followed exact same thoughts as you are! The only concern I had about it is how then the `danger` color would look a bit off compared to `warning` and `success` right next to it:

<img width="569" alt="image" src="https://user-images.githubusercontent.com/57374379/153292347-f75cd2e1-fa12-4bff-8a9c-9ff64e480900.png">

But probably that's not _that_ critical. @wibjorn what do you think? Can we update the color to make it accessible on dark mode with background medium?

Reply:
It's a shame that danger-base on dark doesn't work as a text color. We could consider updating the `danger-base` property on dark to be
```


 But short of revisiting the palette now, I think using the dark variables will work just fine. They were created for this very purpose.

Reply:
We'd then end update with danger-base, success-base, and warning-dark. Which would be consistent with button.

Reply:
Updated the color value and made all relevant updates in PR. Thanks @homing1 for bringing this up 🙏🏻

Reply:
Looks much better to me! For `warning`, `warning-dark` would work for now, but I think we need to revisit. I just realized that even for buttons, the current `$warning-base` is too light without an outline.

![image](https://user-images.githubusercontent.com/94572161/153306461-0095c071-a635-4a48-aed5-0fd78fcfa4ea.png)